File: templates/python/power_system_deep_interaction_matrix.py
"""power_system_deep_interaction_matrix: 电力系统深化交互气泡矩阵（matrix 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='power system analysis: interaction bubble matrix'):
    return make_template_figure('bubble_matrix', seed=3613, title=title, domain='power system analysis', topic='interaction bubble matrix')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/power_system_deep_interaction_matrix.m
function fig = power_system_deep_interaction_matrix()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('bubble_matrix', 3613, 'power system analysis: interaction bubble matrix', 'power system analysis', 'interaction bubble matrix');
end
